src/rebar3_hex_publish.erl

```diff
@@ -326,6 +331,8 @@ to_map(List) when is_list(List) ->
     maps:from_list(List).
 
 include_files(Name, AppDir, AppDetails) ->
+    _ = maybe_print__checkouts_warnings(AppDir),
```

Comment:
Nit: I suggest to call this `maybe_print_checkouts_warnings` vs `maybe_print__checkouts_warnings`

Reply:
I'm Ok with this, though its' just as you say: cosmetics.

Reply:
"Fixed", rebased and force-pushed.